src/phrases/phrases-en.js

```diff
@@ -38,24 +38,6 @@ const phrases = {
     create: 'Create new',
     isSelectedItem: 'Added',
   },
-  article: {
-    author: 'Author',
```

Comment:
Hvorfor er disse ikke i bruk egentlig?

Reply:
Dei hentes fra ndla/ui. Enten må alle defineres i lokal phrases, eller så må ingen. Disse brukes kun for forhåndsvisning av artikkel og treng dermed ikkje være definert dobbelt.